backend/index.md

```diff
@@ -9,15 +9,9 @@ subject: "backend"
 
 ###### learn how to save information, serve up different pages, and make a blog
 
-The backend is all about storing and persisting information. If you want to save
-a user's preferences, do complicated calculations, or store a lot of
-information, you're going to need a backend.
+The backend is all about storing and persisting information.  It usually consists of three main parts: a server, an application, and a database. If you want to save a user’s preferences, do complicated calculations, or store a lot of information, you’re going to need a backend.
```

Comment:
Can you hard wrap your lines to 80 characters? (i.e., insert a newline if the line length exceeds 80 characters)

There's usually a way to get your editor to do this. You might want to see this stack overflow answer for how to do it in Sublime:

http://stackoverflow.com/questions/12466430/what-is-the-usual-way-to-reformat-paragraph-in-sublime-text-2/17159320#17159320